test/Confluent.Kafka.IntegrationTests/Tests/Consumer_OffsetsForTimes.cs

```diff
@@ -101,18 +117,19 @@ public void Consumer_OffsetsForTimes(string bootstrapServers)
         {
             var producerConfig = new ProducerConfig { BootstrapServers = bootstrapServers };
 
+            var baseTime = 100000;
             var messages = new DeliveryResult<byte[], byte[]>[count];
             using (var producer = new ProducerBuilder<byte[], byte[]>(producerConfig).Build())
             {
-                for (var index = 0; index < count; index++)
+                for (var index = 0; index < count; ++index)
```

Comment:
you can tell from this i didn't write these tests!

Reply:
They're much faster now with the increment first.